[DO NOT MERGE][Visual Refresh] Button updates

mgadewoll opened the pull request:
## Summary

>[!IMPORTANT]
This PR is for QA purposes only (until further notice). This PR will not be merged. Instead the changes will be added to Kibana as part of this weeks [EUI upgrade PR](https://github.com/elastic/kibana/pull/222149).

### Changes

1. **EUI changes**: Adds all Visual Refresh related EUI changes on button components:

- `EuiButton`
- `EuiButtonEmpty`
- `EuiButtonIcon`
- `EuiButtonGroup`
- `EuiFilterGroup`/`EuiFilterButton`

2. **EUI button component API updates:** Updates instances of `EuiFilterButton` usages that were missing `isSelected` prop or that should use the new `isToggle` prop to ensure the expected visual and semantic output as toggle button.

<details><summary><b>Examples of new toggle filter buttons</b></summary>

| before | after |
|---|---|
| ![Screenshot 2025-05-05 at 16 18 13](https://github.com/user-attachments/assets/65dcb4f1-27c2-4e73-8d0e-2702de8c35e3) | ![Screenshot 2025-05-02 at 13 48 28](https://github.com/user-attachments/assets/54ec723b-ff3f-49b7-ad6c-0d804fcd8a82) |
| ![Screenshot 2025-05-05 at 16 19 30](https://github.com/user-attachments/assets/65bcef9a-c06b-46e4-91e1-698a76b1dbc2) | ![Screenshot 2025-05-02 at 14 02 46](https://github.com/user-attachments/assets/bf05d9f9-0eba-4733-87f6-d7abdaa53bfa) |
| ![Screenshot 2025-05-05 at 16 21 50](https://github.com/user-attachments/assets/8f4a6592-9dc0-4b69-8ba5-aca0a3c26f69) | ![Screenshot 2025-05-02 at 14 08 07](https://github.com/user-attachments/assets/34d78faa-4137-4650-a4a9-f71cbde64054) |
</details>


3. **Custom styling updates:** Updates custom styling overrides on EUI button components in selected high-visibility places to ensure expected Visual Refresh output. (This is a first iteration step to ensure expected design. In the future we want to replace custom buttons with proper EUI components)

<details><summary><b>Updated custom button styling</b></summary>

| Solution/Area | After |
|---|---|
| Discover(classic) | ![Screenshot 2025-05-30 at 14 45 01](https://github.com/user-attachments/assets/93a46e2a-4989-4bcb-99f2-5eafa4483629) |
| Discover(ES/QL) | ![Screenshot 2025-05-30 at 14 45 11](https://github.com/user-attachments/assets/8fc390d4-fdb0-4377-851c-b3835dc9a14e) |
| Dashboard(Create Visualization) | ![Screenshot 2025-05-30 at 14 47 21](https://github.com/user-attachments/assets/a5895309-9b48-494d-b4b4-e91ca41e316b) |
| Maps: Aligned map toolbar button hover  | ![Screenshot 2025-04-30 at 17 19 11](https://github.com/user-attachments/assets/2d467f6b-ab50-46b1-b393-ed1e6c5d5548) |

</details>

mgadewoll committed:
refactor: update filterButton usages

- adds isToggle and Selected for toggle buttons; adds missing isSelected on expander buttons

mgadewoll committed:
chore: upgrade eui packages

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

kibanamachine commented:
<!-- kbn-message-context:project-deploy-job -->
Project deployments require a Github label, please add one or more of `ci:project-deploy-(elasticsearch|observability|security)` and trigger the job through the checkbox again.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/298644)
* Commit: b78866a9ab4e01bb7f7cedd5c56d3ac133686719

### Failed CI Steps
* [Build Storybooks](https://buildkite.com/elastic/kibana-pull-request/builds/298644#0196a0c5-7cbc-49ec-92a6-25cae7ff0514)

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298644","number":298644,"commit":"b78866a9ab4e01bb7f7cedd5c56d3ac133686719"}],"number":298644}
buildkite-pr-comment-->

mgadewoll committed:
refactor: update custom button styles to align with Visual Refresh button updates

kibanamachine commented:
<!-- kbn-message-context:project-deploy-job -->
PR Project deployment started at: https://buildkite.com/elastic/kibana-deploy-project-from-pr/builds/447

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/298679)
* Commit: d6415d6c1bb111178120d13dc34598209665a0eb

### Failed CI Steps
* [Build and Deploy to Cloud](https://buildkite.com/elastic/kibana-pull-request/builds/298679#0196a0fd-b97a-4531-b785-f46a23843ab4)

### History
* :broken_heart: [Build #298644](https://buildkite.com/elastic/kibana-pull-request/builds/298644) failed b78866a9ab4e01bb7f7cedd5c56d3ac133686719

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298679","number":298679,"commit":"d6415d6c1bb111178120d13dc34598209665a0eb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298644","number":298644,"commit":"b78866a9ab4e01bb7f7cedd5c56d3ac133686719"}],"number":298679}
buildkite-pr-comment-->

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
Cloud deployment initiated, see credentials at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/173

kibanamachine commented:
<!-- kbn-message-context:project-deploy-job -->
Project deployed, see credentials at: https://buildkite.com/elastic/kibana-deploy-project-from-pr/builds/447

kibanamachine commented:
<!-- kbn-message-context:project-deploy-job -->
PR Project deployment started at: https://buildkite.com/elastic/kibana-deploy-project-from-pr/builds/449

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/298926)
* Commit: 3f1a6f90627c0cdabedf415be88b8e83cb5efb62

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/298926#0196a4cb-1f6f-4b8e-8407-11b86f4b9ca9)

### History
* :broken_heart: [Build #298679](https://buildkite.com/elastic/kibana-pull-request/builds/298679) failed d6415d6c1bb111178120d13dc34598209665a0eb
* :broken_heart: [Build #298644](https://buildkite.com/elastic/kibana-pull-request/builds/298644) failed b78866a9ab4e01bb7f7cedd5c56d3ac133686719

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298926","number":298926,"commit":"3f1a6f90627c0cdabedf415be88b8e83cb5efb62"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298679","number":298679,"commit":"d6415d6c1bb111178120d13dc34598209665a0eb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298644","number":298644,"commit":"b78866a9ab4e01bb7f7cedd5c56d3ac133686719"}],"number":298926}
buildkite-pr-comment-->

mgadewoll committed:
test: update tests and snapshots

kibanamachine commented:
<!-- kbn-message-context:project-deploy-job -->
PR Project deployment started at: https://buildkite.com/elastic/kibana-deploy-project-from-pr/builds/451

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
PR Cloud deployment started at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/175

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/298936)
* Commit: e21887c12d917a019159b56537e18f7bf88276ac

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/298936#0196a4e3-def3-44d2-b6b9-c528c6e6b748)

### History
* :broken_heart: [Build #298926](https://buildkite.com/elastic/kibana-pull-request/builds/298926) failed 3f1a6f90627c0cdabedf415be88b8e83cb5efb62
* :broken_heart: [Build #298679](https://buildkite.com/elastic/kibana-pull-request/builds/298679) failed d6415d6c1bb111178120d13dc34598209665a0eb
* :broken_heart: [Build #298644](https://buildkite.com/elastic/kibana-pull-request/builds/298644) failed b78866a9ab4e01bb7f7cedd5c56d3ac133686719

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298936","number":298936,"commit":"e21887c12d917a019159b56537e18f7bf88276ac"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298926","number":298926,"commit":"3f1a6f90627c0cdabedf415be88b8e83cb5efb62"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298679","number":298679,"commit":"d6415d6c1bb111178120d13dc34598209665a0eb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298644","number":298644,"commit":"b78866a9ab4e01bb7f7cedd5c56d3ac133686719"}],"number":298936}
buildkite-pr-comment-->

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
Cloud deployment initiated, see credentials at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/175

kibanamachine commented:
<!-- kbn-message-context:project-deploy-job -->
Project deployed, see credentials at: https://buildkite.com/elastic/kibana-deploy-project-from-pr/builds/451

kibanamachine commented:
<!-- kbn-message-context:project-deploy-job -->
PR Project deployment started at: https://buildkite.com/elastic/kibana-deploy-project-from-pr/builds/452

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
PR Cloud deployment started at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/176

mgadewoll committed:
test: update selector

kibanamachine commented:
<!-- kbn-message-context:project-deploy-job -->
Project deployed, see credentials at: https://buildkite.com/elastic/kibana-deploy-project-from-pr/builds/452

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
PR Cloud deployment started at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/177

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
Cloud deployment initiated, see credentials at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/177

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/299001)
* Commit: 7b5ca85eaa687807fa0ab4ab47db0d0edd7a71ef
* [Cloud Deployment](https://kibana-pr-220093.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #298972](https://buildkite.com/elastic/kibana-pull-request/builds/298972) failed ae36761d5ef336e9690bb6e21de060f66adee843
* :broken_heart: [Build #298936](https://buildkite.com/elastic/kibana-pull-request/builds/298936) failed e21887c12d917a019159b56537e18f7bf88276ac
* :broken_heart: [Build #298926](https://buildkite.com/elastic/kibana-pull-request/builds/298926) failed 3f1a6f90627c0cdabedf415be88b8e83cb5efb62
* :broken_heart: [Build #298679](https://buildkite.com/elastic/kibana-pull-request/builds/298679) failed d6415d6c1bb111178120d13dc34598209665a0eb
* :broken_heart: [Build #298644](https://buildkite.com/elastic/kibana-pull-request/builds/298644) failed b78866a9ab4e01bb7f7cedd5c56d3ac133686719

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/299001","number":299001,"commit":"7b5ca85eaa687807fa0ab4ab47db0d0edd7a71ef"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298972","number":298972,"commit":"ae36761d5ef336e9690bb6e21de060f66adee843"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298936","number":298936,"commit":"e21887c12d917a019159b56537e18f7bf88276ac"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298926","number":298926,"commit":"3f1a6f90627c0cdabedf415be88b8e83cb5efb62"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298679","number":298679,"commit":"d6415d6c1bb111178120d13dc34598209665a0eb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298644","number":298644,"commit":"b78866a9ab4e01bb7f7cedd5c56d3ac133686719"}],"number":299001}
buildkite-pr-comment-->

jeramysoucy committed:
Updates Kibana security custom style components

jeramysoucy commented on src/platform/packages/shared/kbn-user-profile-components/src/user_profiles_selectable.tsx:
```diff
@@ -383,12 +383,7 @@ export const UserProfilesSelectable = <Option extends UserProfileWithAvatar | nu
-                        onClick={() => onChange?.([])}
-                        style={{ height: '1rem' }}
-                      >
+                      <EuiButtonEmpty size="xs" flush="right" onClick={() => onChange?.([])}>
```
Removed '1rem' height - no difference in render.

jeramysoucy commented on x-pack/platform/packages/private/security/ui_components/src/kibana_privilege_table/feature_table.tsx:
```diff
@@ -353,7 +353,7 @@ export class FeatureTable extends Component<Props, State> {
           },
         })}
-        style={{
+        css={{
```
Replaced style with prefered css property. `minWidth` is required to properly render the feature access level options (all, read, none) in the feature table.

jeramysoucy commented on x-pack/platform/packages/shared/security/api_key_management/src/components/token_field.tsx:
```diff
@@ -36,10 +36,9 @@ export const TokenField: FunctionComponent<TokenFieldProps> = ({ value, ...props
       })}
       value={value}
-      style={{
+      css={{
```
Replaced style with preferred css property. Removed `backgroundColor: 'transparent'` - no difference in render.

jeramysoucy commented on x-pack/platform/plugins/shared/security/public/account_management/user_profile/user_profile.tsx:
```diff
@@ -630,7 +630,7 @@ function UserAvatarEditor({
                     size="xs"
                     flush="right"
-                    style={{ height: euiTheme.base }}
+                    css={{ height: euiTheme.base }}
```
Replaced style with prefered css property. `height: euiTheme.base` required to properly render the color picker without misaligning the row.

jeramysoucy commented on x-pack/platform/plugins/shared/spaces/public/space_list/space_list_internal.tsx:
```diff
@@ -107,22 +107,14 @@ export const SpaceListInternal = ({
-          onClick={() => setIsExpanded(false)}
-          style={{ alignSelf: 'center' }}
-        >
+        <EuiButtonEmpty size="xs" onClick={() => setIsExpanded(false)}>
```
Removed style - no difference in render.

jeramysoucy commented on x-pack/platform/plugins/shared/spaces/public/space_list/space_list_internal.tsx:
```diff
@@ -107,22 +107,14 @@ export const SpaceListInternal = ({
-          onClick={() => setIsExpanded(true)}
-          style={{ alignSelf: 'center' }}
-        >
+        <EuiButtonEmpty size="xs" onClick={() => setIsExpanded(true)}>
```
Removed style - no difference in render.

jeramysoucy commented:
@mgadewoll I addressed the Kibana Security custom button style overrides in 9f178b4f1fa8e673929ca68443c4d8435896336a.

cc @SiddharthMantri for review

mgadewoll committed:
refactor: update unified search filter button group styles

- reverts previous changes from primary to text color

- updates border styles from box-shadow to border

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/303628)
* Commit: d3fe6cad0852a423e826c75d3cb0386cda89a1aa

### Failed CI Steps
* [Build Kibana Distribution](https://buildkite.com/elastic/kibana-pull-request/builds/303628#0197208d-a881-42bb-823e-4974fd81ca5d)

### History
* :broken_heart: [Build #303622](https://buildkite.com/elastic/kibana-pull-request/builds/303622) failed d3fe6cad0852a423e826c75d3cb0386cda89a1aa
* :yellow_heart: [Build #299001](https://buildkite.com/elastic/kibana-pull-request/builds/299001) was flaky 7b5ca85eaa687807fa0ab4ab47db0d0edd7a71ef
* :broken_heart: [Build #298972](https://buildkite.com/elastic/kibana-pull-request/builds/298972) failed ae36761d5ef336e9690bb6e21de060f66adee843
* :broken_heart: [Build #298936](https://buildkite.com/elastic/kibana-pull-request/builds/298936) failed e21887c12d917a019159b56537e18f7bf88276ac
* :broken_heart: [Build #298926](https://buildkite.com/elastic/kibana-pull-request/builds/298926) failed 3f1a6f90627c0cdabedf415be88b8e83cb5efb62

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303628","number":303628,"commit":"d3fe6cad0852a423e826c75d3cb0386cda89a1aa"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303622","number":303622,"commit":"d3fe6cad0852a423e826c75d3cb0386cda89a1aa"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/299001","number":299001,"commit":"7b5ca85eaa687807fa0ab4ab47db0d0edd7a71ef"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298972","number":298972,"commit":"ae36761d5ef336e9690bb6e21de060f66adee843"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298936","number":298936,"commit":"e21887c12d917a019159b56537e18f7bf88276ac"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298926","number":298926,"commit":"3f1a6f90627c0cdabedf415be88b8e83cb5efb62"}],"number":303628}
buildkite-pr-comment-->

jughosta commented on src/platform/packages/shared/kbn-unified-data-table/src/utils/get_render_cell_value.test.tsx:
@l-suarez Also, the "+" button for adding filters might be too dark now and steels the attention from other elements on the page.

<img width="948" alt="Screenshot 2025-05-30 at 11 58 47" src="https://github.com/user-attachments/assets/a5342f6f-2696-4b08-9c65-37c9d6845b6b" />

jughosta replied:
Nice! Thanks, @mgadewoll!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/303664)
* Commit: a553e493026cd8704fca337a17b5c865b4073678

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/303664#019720fc-c12f-4d76-a91e-bfcb97ad7f74)

### History
* :broken_heart: [Build #303628](https://buildkite.com/elastic/kibana-pull-request/builds/303628) failed d3fe6cad0852a423e826c75d3cb0386cda89a1aa
* :broken_heart: [Build #303622](https://buildkite.com/elastic/kibana-pull-request/builds/303622) failed d3fe6cad0852a423e826c75d3cb0386cda89a1aa
* :yellow_heart: [Build #299001](https://buildkite.com/elastic/kibana-pull-request/builds/299001) was flaky 7b5ca85eaa687807fa0ab4ab47db0d0edd7a71ef
* :broken_heart: [Build #298972](https://buildkite.com/elastic/kibana-pull-request/builds/298972) failed ae36761d5ef336e9690bb6e21de060f66adee843
* :broken_heart: [Build #298936](https://buildkite.com/elastic/kibana-pull-request/builds/298936) failed e21887c12d917a019159b56537e18f7bf88276ac

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303664","number":303664,"commit":"a553e493026cd8704fca337a17b5c865b4073678"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303628","number":303628,"commit":"d3fe6cad0852a423e826c75d3cb0386cda89a1aa"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/303622","number":303622,"commit":"d3fe6cad0852a423e826c75d3cb0386cda89a1aa"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/299001","number":299001,"commit":"7b5ca85eaa687807fa0ab4ab47db0d0edd7a71ef"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298972","number":298972,"commit":"ae36761d5ef336e9690bb6e21de060f66adee843"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/298936","number":298936,"commit":"e21887c12d917a019159b56537e18f7bf88276ac"}],"number":303664}
buildkite-pr-comment-->

mgadewoll committed:
chore: update eui package versions

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
PR Cloud deployment started at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/214

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
PR Cloud deployment started at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/220

mgadewoll committed:
Merge branch 'main' into eui/visual-refresh-button-updates

mgadewoll commented:
ℹ️ This PR will be added to Kibana as part of this weeks [EUI upgrade PR](https://github.com/elastic/kibana/pull/222149).
